benches/benches/db_lookup_times_utils/matrix.rs

```diff
@@ -0,0 +1,13 @@
+pub const BLOCK_COUNT_MATRIX: [u32; 2] = [10, 100];
+pub const TX_COUNT_MATRIX: [u32; 2] = [100, 1000];
+
+// todo: we can make this lazy loaded
```

Comment:
Is this a stale comment? I assume since the matrix() function returns an iterator that it is lazy loaded already.

Reply:
we call matrix() multiple times, and it was my understanding that it is *not* lazy loaded, like using `OnceCell` perhaps. It should be okay for now given our matrix is small, and will not grow

Reply:
Oh then I see what you mean. Yeah that's probably not the bottleneck. Even if the matrix grows, the iteration should still not be significant in comparison to the benchmarks being run at each iteration. So from that perspective I'd prefer if we just skipped this comment altogether then.

Reply:
agreed 🤝

Reply:
Nice, let me know when you've updated this and I'll approve :rocket:

Reply:
addressed in 0af85338c